toolchain.mk

```diff
@@ -28,12 +33,16 @@ define dltc
 endef
 
 .PHONY: toolchains
-toolchains: aarch32 aarch64
+toolchains: aarch32 aarch32-none aarch64
```

Comment:
Only `synquacer.mk` needs the `none` toolchain, so it may be wiser to add the dependency in `synquacer.mk` rather than `toolchain.mk`.

Reply:
good idea. will do.